src/main/java/frc/lib/controllers/position/PositionControllerSim.java

```diff
@@ -30,9 +30,14 @@ public void configure() {}
 
   @Override
   public void getUpdatedVals(PositionControllerValues values) {
-    values.position = position;
-    values.velocity = velocity;
-    values.motorVoltage = (voltageSet) ? setVoltage : Volts.of(0.0);
+    values.position.mut_replace(position);
```

Comment:
Open to discussion on if the simulation's local state (**position**, **velocity**, **setVoltage**) should be **MutableMeasure**s as well, since there may be multiple allocations per periodic.

Reply:
It seems like that is what **MotorOutputFlywheelSim.java** does, so it should probably be done here as well.